sflow/agent.go

```diff
@@ -91,19 +95,166 @@ func (sfa *Agent) feedFlowTable() {
 		p := gopacket.NewPacket(buf[:n], layers.LayerTypeSFlow, gopacket.DecodeOptions{NoCopy: true})
 		sflowLayer := p.Layer(layers.LayerTypeSFlow)
 		sflowPacket, ok := sflowLayer.(*layers.SFlowDatagram)
+
 		if !ok {
 			logging.GetLogger().Errorf("Unable to decode sFlow packet: %s", p)
 			continue
 		}
 
 		if sflowPacket.SampleCount > 0 {
-			logging.GetLogger().Debugf("%d sample captured", sflowPacket.SampleCount)
 			for _, sample := range sflowPacket.FlowSamples {
 				// iterate over a set of Packets as a sample contains multiple
 				// records each generating Packets.
 				sfa.FlowTable.FeedWithSFlowSample(&sample, bpf)
 			}
+
+			// SFlow Counter Samples
+			Countersamples := sflowPacket.CounterSamples
```

Comment:
I would prefer to not push raw countersamples some fields are not usable (str) and there are not filtered like what you did regarding maxint

Reply:
Ohk, so rather then adding whole Counter Sample in to metadata, it will be good to handle it properly and just add the useful fields into metadata??

Reply:
Yes adding only useful metadata will be better. I think this could be addressed in another PR. For this one I would simply remove it for now but it is up to you provide a structure :)

Reply:
Ohk, we are removing it and do it in another PR.
Thanks!

Reply:
I added the structure but as of now, not adding PortName and LACP Counter Record types.